.github/workflows/release.yml

```diff
@@ -41,3 +41,6 @@ jobs:
         if: always()
         run: |
           rm -fv minisign.key
+  upload:
+    needs: release
+    uses: ./.github/workflows/upload.yml
```

Comment:
I think this is the syntax to call another workflow but I haven't used this myself before.